changelog.txt

```diff
@@ -8,7 +8,7 @@ V.NEXT
 - [MINOR] Fix issue for MSA only where headers are not propagated on web requests with different domain redirects on newer versions of WebView (88+) (#2072)
 - [PATCH] Add method to return list of broker that supports account manager (#2073)
 - [MINOR] Send client id as part of request bundle for getSsoToken Api (#2064)
-- [PATCH] Wire new Broker Discovery Client into MSAL - still disabled by default (#2057)
+- [MAJOR] Wire new Broker Discovery Client into MSAL - still disabled by default (#2057)
```

Comment:
is this change intentional? Changing PATCH to MAJOR?

Reply:
yes this is intentional.